### ISSUE TITLE
Feedback on FINRA FIRST requested

### ISSUE THREAD

**Issue by jgavronsky** [finos/open-regtech-sig#50]
## FINRA needs your input (by February 21, 2023)

Dear Open RegTech community, 
As you know, FINRA has presented to this community about their new rule taxonomy tool called [FIRST](https://www.finra.org/rules-guidance/rulebooks/finra-rules). They would like to receive comments from the community on any and all questions they have posed. Please take 5 minutes and [send a response](https://www.finra.org/rules-guidance/notices/special-notice-102122#noticenmbnb), if you are able. This will help FINRA decide how to proceed with this effort.

Alternatively, if you are pressed for time, we ask that you vote here, so FINOS can submit a collective response from this community. At a minimum, just give this issue a "thumbs-up" 👍  or a "thumbs-down" 👎 here.

Ideally, please consider also adding comments, here are some specific questions, but any comments are welcome:
- Is FIRST a useful tool for the industry and will you and/or your firm use it?
- Should FINRA continue to develop the taxonomy
- Is the API useful and do/will you or your firm use it?
- Should FINRA open-source the rule book and continue development of the taxonomy by continuing collaboration in the open?
- If development were done in the open, will you engage in this process (directly or via feedback)?

Thank you!

**Comment by nicholasvitalo** [comment]
Nick Vitalo from FINRA, again.  Thank you again for your time, all.  [Here](https://www.finra.org/media-center/finra-unscripted/machine-readable-rulebook-initiative) is a link to a podcast that our project leads did that explains the initiative at  greater depth.  In addition, in case I was unclear earlier, the Special Notice includes specific questions on which FINRA is seeking comment - but we are equally interest in any other areas of feedback that folks may have.

**Comment by alexk2021** [comment]
Thank you.  As noted, the comment period for the [FINRA Rulebook Search Tool (FIRST)](https://www.finra.org/media-center/finra-unscripted/machine-readable-rulebook-initiative) is open until February 21st.  We'd also like to highlight two recent podcasts with:
* [FINRA Unscripted](https://www.finra.org/media-center/finra-unscripted/machine-readable-rulebook-initiative), and
* [Alliance for Innovative Regulation (AIR)](https://regulationinnovation.org/podcast/machines-that-read-regulations-finras-goal-could-soon-be-reality/)

**Comment by jgavronsky** [comment]
I would like to offer the following comments to the [FINRA Notice](https://www.finra.org/rules-guidance/notices/special-notice-102122#notice) and in response to the questions above.

- 👍 Is FIRST a useful tool for the industry and will you and/or your firm use it?

_Yes, while FINOS’s work is not directly linked to determining compliance with FINRA rules, I believe this is an extremely useful tool for the industry._

- ✅ Should FINRA continue to develop the taxonomy

_Yes, FIRST should be further expanded, to include more rules, as well as to compare and rationalize the resulting taxonomy with other taxonomies developed by vendors and market participants._

- 🆒 Is the API useful and do/will you or your firm use it?

_Having an API for digitized access is absolutely critical for efficient use of the rule book by market participants_

- ❤️ Should FINRA open-source the rule book and continue development of the taxonomy by continuing collaboration in the open?

_Yes, absolutely. Open collaboration is the best way to:_

_1. Solicit input from the best experts across the industry_
_2. Deliver a comprehensive taxonomy and rule applicability model, which can be further extended to incorporate rules from other regulators and jurisdictions_
_3. Ensure adoption and use of the tool and its API_
_4. Reduce the cost burden borne by a single entity_
_5. Create transparency and equal access for market participants through a fully accessible digital platform_
_6. Enable the growth of innovative approaches and new businesses on the basis of this work_

- 🏃 If development were done in the open, will you engage in this process (directly or via feedback)?

_Yes, absolutely. FINOS would be very happy to continue to leverage our Open RegTech Strategic initiative, our community channels, and our experience in creating and nurturing open standards and open source projects to assist in any further steps that engage in collaborative discussions in the open and contributions to open-license repositories. Furthermore, we can continue to gather the industry practitioners to provide feedback and input on this work.



In addition to the above summary, I would like to offer the following thoughts to explain the reasoning behind some of my answers above.

### Digitization of Regulation rules and FINOS’s Open RegTech strategic vision
Digitization of the regulatory rule books is essential for efficient and comprehensive oversight, and equally so for enabling safe market development and innovation. FINOS is convening participants from the financial services community who want to drive game-changing innovations in the practice of regulation through technology. We believe that digitization of regulations, and specifically authoring of regulations in “digital-first” form will significantly simplify the burden of compliance, increase precision and speed of compliance, and will materially cut the cost of compliance, all the while creating faster access to comprehensive information and enabling more nimble response times to market events.
With this vision in mind, we recognize that the road to get to full digital-first model is a long one, and today we have many existing regulations which did not originate in digital form and whose automation (in some form) can bring great benefits to market participants. This is where continuation of the current industry (and FINRA’s) efforts around rule taxonomies can help significantly.  Taxonomies will help in digitizing the current rule books, and will enable both regulators and market participants to navigate the regulations with greater speed and clarity.  They will also pave the way for streamlined, common taxonomy models to become standards across the industry and eventually to enable issuance of regulations in digital-first form.

Harmonization or standardization of taxonomies is a natural next step for the industry, and the input and advice FINRA’s Taxonomy Working Group provided can deliver even greater impact across a bigger number of participants by inviting feedback and collaboration in the open.  To this end, the open collaboration process and governance FINOS has established and practiced can be leveraged to continue convening industry participants and incorporate contributions from regulated entities of all sizes, as well as from RegTech vendors and technology providers. 

### Open-sourcing the FINRA rule book taxonomy and its API

There are several compelling reasons for FINRA to open source its rule book taxonomy and API, as well as to collaborate with industry participants and other regulators. A taxonomy which is developed openly under an open source license will **_promote collaboration with industry participants_**. By working together with market participants, FINRA can gain a _**deeper understanding of the challenges and opportunities**_ facing the industry, as well as identify areas where regulation can be improved. This can lead to more **_effective and efficient regulation_**, as well as a more vibrant and innovative industry. Furthermore, _**increased transparency and accountability**_ will help to promote _**better understanding of the regulations and rules**_ that govern the securities industry, and can ultimately lead to a _**more efficient and effective marketplace**_.

_**Collaboration among regulators**_ is also essential to ensure that the securities industry is regulated effectively and will help to avoid inconsistencies and overlaps in regulation, by providing a common platform for information sharing and analysis.
In addition, open sourcing FINRA's rule book taxonomy and API will also _**directly drive innovation**_ in financial technology by encouraging development of new and innovative solutions to the challenges facing the securities industry. This can help to drive growth and competitiveness in the industry, as well as promote the adoption of new technologies and best practices.

### FINOS is the place for open collaboration in Financial Services

FINOS has a strong track record of convening the community to develop industry standards. For example, FINOS currently hosts the [Financial Desktop Connectivity and Collaboration Standard (FDC3)](https://fdc3.finos.org/), which aims to standardize interaction and data exchange among financial desktop applications. FINOS [recently launched](https://www.accesswire.com/viewarticle.aspx?id=739516) the project for open development of the [Common Data Model (CDM)](https://www.finos.org/common-domain-model), which standardizes the definition of financial products and workflows.
The [FINOS Open RegTech strategic initiative](https://www.finos.org/open-regtech) is well suited to bring the community together around the development of a rule book taxonomy for FINRA, and eventually for other regulations. This initiative is focused on fostering innovation in the regulatory technology space, and provides a platform for collaboration and cooperation among stakeholders in the industry. Through this initiative, FINOS can bring together a diverse group of experts, including financial institutions, technology companies, regulators, and academics, to work on developing a comprehensive and accessible taxonomy for the FINRA rule book.

In conclusion, FINOS and its community are well positioned to facilitate the open collaboration that is necessary for FINRA to open source its rule book taxonomy and API. FINOS has a strong track record of convening the community to develop industry standards, and its Open RegTech initiative is best suited to bring the community together around the development of a rule book taxonomy for FINRA and other regulations. By leveraging the expertise and resources of the FINOS community, FINRA can help to promote transparency, accountability, and innovation in the securities industry.

I invite others in our community to offer thoughts and suggestions here, and we can host additional discussions during our upcoming SIG meetings.
